src/NServiceBus.Core.Tests/Scheduler/ScheduleTests.cs

```diff
@@ -73,7 +73,7 @@ public Task Publish(object message, PublishOptions options)
                 throw new NotImplementedException();
             }
 
-            public Task Publish<T>(Action<T> messageConstructor, PublishOptions publishOptions)
+            public Task Publish<T>(Action<T> messageConstructor, PublishOptions options)
```

Comment:
Given https://github.com/Particular/NServiceBus/pull/5772 should we roll this back?